.github/workflows/pull-request-master.yml

```diff
@@ -104,6 +104,7 @@ jobs:
           go-cache-dep-path: "**/go.sum"
           go-version-file: go.mod
           go-test-cmd: make test-ci
+          enable-go-test-race: "true"
```

Comment:
Is this the line that will be run?
https://github.com/smartcontractkit/.github/blob/main/actions/ci-test-go/action.yml#L75

Can we update that with the `GORACE` env var set to create separate files for race traces? Otherwise they are mixed in with test logs and can be difficult to find. In core we add this: `GORACE="log_path=$PWD/race"`

Reply:
Makes sense. It should be all set.